src/main/java/baritone/process/elytra/LegacyElytraBehavior.java

```diff
@@ -110,8 +99,8 @@ public final class ElytraBehavior extends Behavior implements IElytraBehavior, H
     private final int[] nextTickBoostCounter;
 
     private BlockStateInterface bsi;
-    private BlockStateOctreeInterface boi;
-    private BlockPos destination;
+    private final BlockStateOctreeInterface boi;
+    public BlockPos destination; // TODO: make this final?
```

Comment:
yeah this should be final. `pathTo` is always called on a new legacy behavior instance